docs/documentation/contributing.md

```diff
@@ -87,12 +87,13 @@ be assigned by the Technical Board or Maintainers:
 |-----------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | bugfix          | PRs with the label `bugfix` propose a solution for a reported bug in the official Bugtracker https://mantis.ilias.de                                                      |
 | improvement     | The label `improvement` is used for PRs which propose a general improvement of code or documentation which is not related to a bug.                                       |
+| roadmap         | The label `roadmap` is assigned to PRs that contain strategical or tactical discussions regarding the future of a component.                                              |
```

Comment:
strategical or tactical discussions **of technical topics** regarding the future of a component.